webapp/src/app/actions.ts

```diff
@@ -66,6 +66,25 @@ export const logoutThunk = createAsyncThunk<
   return response as RequestResponse<null>;
 });
 
+export const fetchDogsThunk = createAsyncThunk(
+  "fetchAllDogs",
+  async (item: any, { rejectWithValue }) => {
```

Comment:
Add type to "item", preferably just simple 'type' so it can be imported

Reply:
![image](https://user-images.githubusercontent.com/27913211/114405945-6cac3180-9ba7-11eb-8fc6-6f72230bc635.png)
just like in config at the top